src/types/system-module.ts

```diff
@@ -0,0 +1,20 @@
+import { AppUserInterface } from './app';
+
+export interface SystemModuleInterface {
+  id: string;
+  uuid: string;
+  name: string;
+  slug: string;
+  model_name: string;
+  app: AppUserInterface;
+  parent: SystemModuleInterface;
+  menu_order: number;
+  show: number;
+}
+export interface Where {
```

Comment:
Suggestion: we already have a where condition named "WhereCondition" type that includes `column, operator, and value` you can modify that to get the "and" and de "or" but this argument should be optional